.ci/docker/build.sh

```diff
@@ -361,7 +361,6 @@ fi
 # Build image
 DOCKER_BUILDKIT=1 docker build \
        --no-cache \
-       --progress=plain \
```

Comment:
Just curious, why this is needed?

Reply:
Danylo added it, I did not remove. I can try without it.

@DanilBaibak what is the reasoning here?